content/news/release-20180218.md

```diff
@@ -0,0 +1,56 @@
+
+The changes weren't all behind the scenes; an overhaul of the multiplayer UI added long-requested features:
+
+ * The server browser shows the name/faction/team of connected players
+ * The server browser is now also available as a tab in the server lobby
+ * The main menu and server browser will notify you if a new release is available
+ * The Global Chat has been removed from the server browser and lobby
+ * A chat dialog has been added to the in-game menu
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20180102-serverplayers.png" width="600" alt="Server player list">
+
+The new server player list is one of several improvements to the multiplayer UI.
+</div>
+
+Of course, we realize that the vast majority of OpenRA players do not play online. We haven't forgotten about you either!
+Improvements for the singleplayer experience include:
+
+ * The full Harkonnen campaign in D2K
+ * Refinements and improvements to several campaign missions in Red Alert and Tiberian Dawn
+ * A fix for the AI cheating when it uses superweapons and support powers
+ * Several other changes that should improve the AI behaviour and performance
+
+<div style="text-align:center" markdown="1">
+![AI Airstrike](/images/news/20180102-airstrike.png)
+
+Improvements to the AI support power targeting mean that this will be a rarer sight.
+ </div>
+
+Other notable changes include:
+
+* New lobby options to disable bounty, build radius, and re-packing Construction Yards
+* Units will no longer target enemies revealed by the Red Alert GPS Satellite
+* A collection of community-driven balance changes for Red Alert
+* Fixes for many graphical glitches and polish issues
+* Fixes for several platform integration issues on Windows and macOS
+* Further improvements to memory usage and performance
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20180102-mapoptions.png" width="600" alt="Expanded game options">
+
+The Map Options tab includes several new settings, and can be customized by scripted maps.
+</div>
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20180102-winmacos.png" width="600" alt="Windows and macOS improvements">
+
+Platform integration fixes include "Pin to taskbar" support on Windows and 64 bit support for macOS.
+</div>
+
+As always, we suggest you read the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/1277bf3eb2b17e29fe9f2f6fd658c90771d7edbe) if you'd like more information on the full list of changes.
+Let us know what you think and please report any bugs you discover via [our forum](http://www.sleipnirstuff.com/forum/viewforum.php?f=80), our [GitHub issue tracker](https://github.com/OpenRA/OpenRA/issues), or in the comments below.
```

Comment:
We usually don't include this on release posts, but instead point to the download page again

Reply:
Something like "...if you'd like more information on the full list of changes, or go directly to our [download page](link) to try them out for yourself!"